src/frontend/screens/Game/GamePage/components/MainButton.tsx

```diff
@@ -62,6 +74,48 @@ const MainButton = ({ gameInfo, handlePlay, handleInstall }: Props) => {
     )
   }
 
+  function showAltPlayAction() {
+    if (
+      is.syncing ||
+      is.installingRedist ||
+      is.installingWinetricksPackages ||
+      is.launching ||
+      is.playing
```

Comment:
Maybe factor this out into its own variable? Seems like this condition (although with some variation on which `is` values are considered) is used 3 times now